src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Configuration/GeneralSettings.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Authorization.Configuration
+{
+    /// <summary>
+    /// General configuration settings
+    /// </summary>
+    public class GeneralSettings
+    {
+        /// <summary>
+        /// Gets or sets the sbl cookie name
+        /// </summary>
+        public string SBLCookieName { get; set; }
```

Comment:
Do we actually need the SBLCookieName in these settings?

Reply:
i actually dont know, i thought may be we need when integration testing

Reply:
Nop. Only the bridge endpoint should be required in this component as far as I can see :)

Reply:
fixed